src/core/client/admin/test/community/community-rtl.spec.tsx

```diff
@@ -0,0 +1,545 @@
+  const siteModeratorViewer = users.moderators[1];
+  const siteModeratorUser = users.moderators[2];
+
+  const resolvers = createResolversStub<GQLResolver>({
+    Query: {
+      viewer: () => siteModeratorViewer,
+      settings: () => settingsWithMultisite,
+    },
+    Mutation: {
+      demoteModerator: ({ variables }) => {
+        expectAndFail(variables).toMatchObject({
+          userID: siteModeratorUser.id,
+          siteIDs: [sites[0].id],
+        });
+        const userRecord = pureMerge<typeof siteModeratorUser>(
+          siteModeratorUser,
+          {
+            moderationScopes: { scoped: false, sites: [] },
+          }
+        );
+        return {
+          user: userRecord,
+        };
+      },
+    },
+  });
+  await createTestRenderer({
+    resolvers,
+  });
+
+  const userRow = await screen.findByTestId(
+    `community-row-${siteModeratorUser.id}`
+  );
+  const changeRoleButton = within(userRow).getByLabelText("Change role");
+  userEvent.click(changeRoleButton);
+
+  const popup = within(userRow).getByLabelText(
+    "A dropdown to promote/demote a user to/from sites"
+  );
+  const siteModButton = within(popup).getByRole("button", {
+    name: "Remove my sites from moderator",
+  });
+  siteModButton.click();
+
+  await waitFor(() => screen.getByTestId("siteModeratorActions-modal"));
+  const removeButton = screen.getByRole("button", { name: "Remove" });
+  removeButton.click();
+
+  expect(resolvers.Mutation!.demoteModerator!.called).toBe(true);
+});
+
+it("load more", async () => {
+  await createTestRenderer({
+    resolvers: createResolversStub<GQLResolver>({
+      Query: {
+        users: ({ callCount }) => {
+          switch (callCount) {
+            case 0:
+              return {
+                edges: [
+                  {
+                    node: adminViewer,
+                    cursor: adminViewer.createdAt,
+                  },
+                  {
+                    node: users.commenters[0],
+                    cursor: users.commenters[0].createdAt,
+                  },
+                ],
+                pageInfo: {
+                  endCursor: users.commenters[0].createdAt,
+                  hasNextPage: true,
+                },
+              };
+            default:
+              return {
+                edges: [
+                  {
+                    node: users.commenters[1],
+                    cursor: users.commenters[1].createdAt,
+                  },
+                ],
+                pageInfo: {
+                  endCursor: users.commenters[1].createdAt,
+                  hasNextPage: false,
+                },
+              };
+          }
+        },
+      },
+    }),
+  });
+
+  const loadMore = screen.getByText("Load More");
+  loadMore.click();
+
+  await waitFor(() =>
+    expect(screen.queryByText("Load More")).not.toBeInTheDocument()
+  );
+
+  // Make sure third user was added.
+  screen.getByText(users.commenters[1].username!);
```

Comment:
I think it would be good to make this an explicit assertion here, so `expect(screen.getByText(users.commenters[1].username!)).toBeVisible();`, so that it's clear quickly what we are asserting.

Reply:
👍